src/components/atoms/Anchor.js

```diff
@@ -43,6 +45,7 @@ const Anchor = ({
   <StyledAnchor
     href={href}
     fontFamily={fontFamily}
+    lineHeight={lineHeight}
```

Comment:
lineHeight by mal bt rovnaky vsade :/

Reply:
To som musel dať kvôli socialButtonu, aby boli zarovno